src/webhint-theme/layout/partials/home.hbs

```diff
@@ -59,18 +59,39 @@
                 </div>
             </section>
         </div>
+        <section class="container home-container--section" id="scanner-input">
+            <div class="layout layout--basic--offset--alt">
+                <div class="module module--primary">
+                    <h1 class="headline">Use the online scanner</h1>
+                    <p class="container--section__welcome">Not sure what you want to configure the CLI to check? Use the online scanner for a pre-configured set of rule checks to cover the basics.</p>
+                    <div class="home-container--input">
+                        <form action="/scanner/" method="POST">
+                            <label for="home-scan" class="visually-hidden">Enter your URL here</label>
+                            <div class="input-fix">
+                                <input id="home-scan" type="url" name="url" placeholder="Type your URL here">
+                                <button class="button--red" type="submit">Run Scan</button>
+                            </div>
+                        </form>
+                    </div>
+                </div>
+                <div class="module module--secondary">
+                    <img src="/images/nellie-searching.svg" alt="" role="presentation">
+                </div>
+            </div>
+        </section>
         <div class="bg-wrap--grey">
-            <section class="container home-container--section home-container__install">
+            <section class="container home-container--section home-container__install" id="cli-howto">
                 <h2 class="subheading subheading--home">Run webhint from your CLI</h2>
-                <div class="layout layout--halves vertical-align">
+                <div class="layout layout--halves">
                     <div class="module">
-                        <img src="images/cli.gif" alt="webhint cli">
+                        <img src="images/hint.gif" alt="webhint cli">
                     </div>
                     <div class="module">
                         <p class="install__code">If you want to disable specific hints, further configure others, ignore domains, or more, run webhint from your CLI.</p>
                         <p class="install__code">Quick start guide:</p>
                         <div class="code-wrap">
                             <p>><code>npm install -g --engine-strict webhint</code></p>
```

Comment:
This should be `npm install -g --engine-strict hint`

Reply:
Maybe we should have the npx command here now that it works? @webhintio/core?